src/org/zaproxy/zap/extension/fuzzdb/files/fuzzers/fuzzdb/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
Are these `.directory` files really needed? I see that those are also in the `fuzzdb` repo.